cmd/list.go

```diff
@@ -0,0 +1,280 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sync"
+	"text/tabwriter"
+
+	"github.com/urfave/cli"
+
+	"github.com/manifoldco/torus-cli/api"
+	"github.com/manifoldco/torus-cli/apitypes"
+	"github.com/manifoldco/torus-cli/config"
+	"github.com/manifoldco/torus-cli/envelope"
+	"github.com/manifoldco/torus-cli/errs"
+	"github.com/manifoldco/torus-cli/pathexp"
+)
+
+func init() {
+	list := cli.Command{
+		Name:      "list",
+		ArgsUsage: "",
+		Usage:     "List your org's project structure and its secrets.",
+		Category:  "SECRETS",
+		Flags: []cli.Flag{
+			orgFlag("Use this organization.", false),
+			projectFlag("Use this project.", false),
+			envSliceFlag("Use this environment.", false),
+			serviceSliceFlag("Use this service.", "", false),
+			nameFlag("Find secrets with this name."),
+			cli.BoolFlag{
+				Name:  "verbose, v",
+				Usage: "Lists the sources of the secrets (shortcut for --format verbose)",
+			},
+		},
+		Action: chain(
+			ensureDaemon, ensureSession, loadDirPrefs, checkRequiredFlags, listCmd,
+		),
+	}
+	Cmds = append(Cmds, list)
+}
+
+func listCmd(ctx *cli.Context) error {
+	verbose := ctx.Bool("verbose")
+
+	args := ctx.Args()
+
+	cfg, err := config.LoadConfig()
+	if err != nil {
+		return err
+	}
+
+	client := api.NewClient(cfg)
+	c := context.Background()
+
+	// TODO - Make this a type
```

Comment:
Is this a todo you're going to resolve ?

Reply:
Yes, have resolved the levels of maps into seperate types, going into the next push.

Reply:
Looks like you forgot to remove the comment!

Reply:
Fixed.